deebotozmo/vacuum_bot.py

```diff
@@ -179,8 +183,6 @@ async def handle(self, event_name: str, event: dict, requested: bool = True) ->
                 await self._handle_charge_state(event_data)
         elif event_name == "lifespan":
             await self._handle_life_span(event_data)
-        elif event_name == "cleanlogs":
```

Comment:
Isn't there a chance that we get a cleanlogs event via mqtt? I don't think this should be removed. I already added a `return` statement to end execution of the rest of the function in fact it was `requested == True`

Reply:
Clean events can only be requested as they are not stored directly on the robot but an the ecovacs servers. Therefore it's fine to move it